addon/ng2/blueprints/ng2/files/src/app.ts

```diff
@@ -1,4 +1,4 @@
-import {bootstrap} from 'angular2/angular2';
+import {bootstrap} from 'angular2/platform/browser';
```

Comment:
also add a link to https://github.com/angular/angular/issues/5596 for more context as to why this line is needed